cypress/integration/auth.js

```diff
@@ -0,0 +1,38 @@
+describe('Auth', () => {
```

Comment:
It'd be great to have a test for signup to team, though might be hard?

Reply:
done!